File: code-playbook/odds_ends/spring-cloud-msa-skeleton_gradle/order-service/src/main/java/com/example/order/repo/OrderRepository.java
package com.example.order.repo;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

import com.example.order.entity.OrderEntity;

public interface OrderRepository extends JpaRepository<OrderEntity, Long> {
    List<OrderEntity> findByUserId(long userId);
}

File: code-playbook/practice_kotlin/kotlin-trading-core/src/main/kotlin/com/revy/trading/adapter/output/persistence/repository/OrderRepository.kt
package com.revy.trading.adapter.output.persistence.repository

import com.revy.trading.adapter.output.persistence.entity.OrderEntity
import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface OrderRepository : JpaRepository<OrderEntity, UUID> {

    fun findByOrderId(orderId: UUID): OrderEntity?
}